### PR TITLE
use autoprefixer

### PR DESCRIPTION
**Pull request by Johann-S**
Fixes: #52

### CODE REVIEW

**Comment by Johann-S** [comment]
Maybe we should think about using a Web worker, I'm not sure but I think it's possible it'll freeze on some older PC 🤔

**Comment by XhmikosR** [comment]
That is true but this shouldn't add so much overload anyway.

Otherwise we go with the alternative way I suggested the other day on Slack.

File: src/js/build.js [hunk 1]
```diff
@@ -2,12 +2,26 @@ import axios from 'axios'
...
 import Sass from 'sass.js/dist/sass'
 import CleanCSS from './lib/clean-css'
+import autoprefixer from './lib/autoprefixer'
 
 import { jsPlugins, scssPlugins } from './plugins'
 import { createJsFileContent, generateLink } from './file-util'
 import { formatScssList, uniqArray } from './util'
 
 const popperCDN = 'https://unpkg.com/popper.js/dist/umd/popper.js'
+const autoPrefixerBrowsers = [
```
Comment (XhmikosR) [inline src/js/build.js]: I'd change this to an object, keep the browsers as an array and add `cascade: false` like we do upstream.

**Comment by XhmikosR** [comment]
Also, can't we load autoprefixer after the page load (not in our main bundle)?

**Comment by Johann-S** [comment]
I tried but it leed me to my other PR...

File: src/js/build.js [hunk 2]
```diff
@@ -2,12 +2,31 @@ import axios from 'axios'
...
+import autoprefixer from './lib/autoprefixer'
 
 import { jsPlugins, scssPlugins } from './plugins'
 import { createJsFileContent, generateLink } from './file-util'
 import { formatScssList, uniqArray } from './util'
 
 const popperCDN = 'https://unpkg.com/popper.js/dist/umd/popper.js'
+const autoPrefixerConfig = {
+  cascade: false,
+  from: undefined,
```
Comment (XhmikosR) [inline src/js/build.js]: Is `from` really needed?

Comment (Johann-S) [reply src/js/build.js]: yep without that we have a warning in the `console`

Comment (XhmikosR) [reply src/js/build.js]: Can it be just an empty string?

Comment (Johann-S) [reply src/js/build.js]: the warning said, to avoid that we should use `undefined`

Comment (XhmikosR) [reply src/js/build.js]: OK, I guess, I just don't get what is throwing the warning :P

Comment (Johann-S) [reply src/js/build.js]: image

Comment (XhmikosR) [reply src/js/build.js]: Ah, OK, makes sense now. All good then.

**Pull request merged**